cmd/kubeadm/app/images/images_test.go

```diff
@@ -315,21 +315,21 @@ func TestGetDNSImage(t *testing.T) {
 		cfg      *kubeadmapi.ClusterConfiguration
 	}{
 		{
-			expected: "foo.io/coredns:v1.11.3",
+			expected: "foo.io/coredns:v1.11.4",
```

Comment:
please make all test files to use the constant.
e.g.

 ```suggestion
			expected: "foo.io/coredns:"+constants.CoreDNSVersion,
```